src/call.cpp

```diff
@@ -3510,6 +3510,10 @@ call::T_ActionResult call::executeAction(char * msg, message *curmsg)
         } else if (currentAction->getActionType() == CAction::E_AT_PAUSE_RESTORE) {
             double operand = get_rhs(currentAction);
             paused_until = (int)operand;
+#ifdef RTP_STREAM
+        } else if (currentAction->getActionType() == CAction::E_AT_RTP_ECHO) {
+            rtp_echo_state = (currentAction->getDoubleValue() != 0 ? true : false);
+#endif
```

Comment:
There is an RTP_STREAM ifdef below on line 3693, why not put it in there.

And `rtp_echo_state = (currentAction->getDoubleValue() != 0)` is explicit enough for a boolean assignment.